docs/docs/kit/haptics.md

```diff
@@ -2,4 +2,11 @@
 
 ```sh
 npm install @rise-tools/kit-haptics expo-haptics
-```
\ No newline at end of file
+```
+
+## Actions
+
+- `haptics`
+- `haptics('impact', type)`
+- `haptics('notification', type)`
+- `haptics('selection', type)`
```

Comment:
I think selection doesn't have a type, does it?

Also we should specify the actual values. Feel free to just add some notes here for the valid values, and I will clean up the docs later

Reply:
Yes, my bad. Updated `selection` type. I will cross-link to Expo for now and let you figure the best way to present it when you get to it!